api/Operation.md

```diff
@@ -0,0 +1,74 @@
+# Operation
+
+## Web Server Configuration
+
+Diem node generates the following default API configuration:
+
+```
+api:
+  enabled: true
+  address: "0.0.0.0:8080"
+```
+
+It's optional to configure TLS certificate and private key file paths for enabling TLS for the web server:
+```
+api:
+  enabled: true
+  address: "0.0.0.0:8080"
+  tls_cert_path: <file path>
+  tls_key_path: <file path>
+```
+
+When `api.enabled` is set to `true`, both API and JSON-RPC configured web server will serve the REST API.
+
+### JSON-RPC is enabled
+
+Regardless what is JSON-RPC `address` configuration, JSON-RPC API is always enabled at the REST API port.
+
+However, JSON-RPC service still reads configurations from `json_rpc` section.
+
+To avoid enabling JSON-RPC and REST API at different port, you can configure same address for `api.address` and
+`json_rpc.address` (you are required to set them same, as well as `tls_cert_path` and `tls_key_path`
+for `api` and `json_rpc`).
+
+
+## Health check endpoint
+
+Health check: `/-/healthy` returns 200
```

Comment:
Is this a typo? URL of `/-/healthy` is awkward.

Reply:
This is not a type and is the existing path for doing a health check

Reply:
Took me a while to figure out what is the URL path when I first saw it.
But it is the correct health check url path. You can curl it local :)
It's kind of legacy from JSON-RPC service, but our production env depends on it, so I just ported it to API as well.